### DISCUSSION TITLE
Migrating from v2 to v3 - running in Jenkins pipeline, how to make a pipeline stage turn red if a check has returned FAIL result?

### DISCUSSION THREAD

**Discussion by LB50000** [prowler-cloud/prowler#1822] [Q&A]
Hi,

I'm moving from v2 to v3 and on v2 when prowler was executed inside a Jenkins pipeline stage like so:
```
            stage('Run') {
                 sh script: "./prowler/prowler -r eu-west-2 -g cislevel2
```

If any checks within this compliance group fail because it's run using an 'sh' block, the stage turns red, likewise if they don't the stage turns green, based on the exit status.

How can I replicate this behaviour in v3?

Thanks in advance.

**Comment by toniblyx** [comment]
Hi @LB50000 in prowler v3, the command you need to run (if you install prowler using pip) is `prowler --filter-region eu-west-2 --compliance cis_1.4_aws`

For you output status we will add a flag to give exit 1 when a FAIL is found like in v2. Coming in the next weeks.
